scripts/python3.8/linux/build_in_docker.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
(Same comment as above: Put in "docker" folder? The comment below would be self-evident then.)

Reply:
Yes, I also agree.